runtime/compiler/runtime/IProfiler.hpp

```diff
@@ -92,6 +92,7 @@ class TR_ResolvedMethod;
 class TR_AbstractInfo;
 class TR_BitVector;
 class TR_J9VMBase;
+class TR_J9SharedCache;
```

Comment:
After TR_J9SharedCache was introduced to IProfiler.hpp, and after JITServerIProfiler.cpp was delivered, we always needed this. I guess it wasn't uncovered because the other places that include IProfiler.hpp also have TR_J9SharedCache included. I should have included this fix in https://github.com/eclipse/openj9/pull/7096 but accidentally removed it. As a result, the current master branch JITServer build is still broken without this forward declaration.